docs/writing-a-locustfile.rst

```diff
@@ -93,12 +94,6 @@ Usually, this is specified in Locust's web UI or on the command line, using the
 If one declares a host attribute in the user class, it will be used in the case when no :code:`--host` 
 is specified on the command line or in the web request.
 
-The *tasks* attribute
----------------------
-
-A User class can have tasks declared as methods under it using the :py:func:`@task <locust.task>` decorator, but one can also
-specify tasks using the *tasks* attribute which is described in more details :ref:`below <tasks-attribute>`.
-
```

Comment:
Why remove this?

Reply:
Mainly because the index on the left hand side looks way better, but mainly because repetition/duplication bad :)

Reply:
I think the "Writing a locustfile" should be a full reference of all important aspects on writing test scripts, so I think we should include it.

Reply:
Uh. my previous comment was actually about why I had removed "the" and "property" from lots of headings. Dont know why it ended up here...

Reply:
The reason for removing was that the tasks attribute is already mentioned under Tasks, no point in repeating it under User.

Reply:
I see. When I wrote it, the idea was to list all the available public attributes, even though a more detailed description exists under Tasks (which the text refers to).

I'm not sure that I prefer this structure:

![image](https://user-images.githubusercontent.com/54217/82425079-57598080-9a86-11ea-8c3c-44c919bfa332.png)

to this:

![image](https://user-images.githubusercontent.com/54217/82425108-5fb1bb80-9a86-11ea-9456-200f75ad4a45.png)

We could definitely drop the "The", but I actually think the second version is more clear.

Reply:
Ok, I'll add back the "attribute". 

TBH I'm a bit frustrated by the amount of time I've had to spend on this PR. I know you're just trying to make things as nice as possible, but maybe next time file your own PR with adjustments, if the gist of my PR is good? Otherwise I dont think I'll bother fixing the docs any more...

Reply:
I'm sorry you feel that way :/, but there were quite a few of the changes to the Quickstart docs that I thought didn't make it better. 

I spent quite a few hours re-writing those docs *very recently* for the 1.0 changes (mostly in these commits: e40edfed6e08966c722e03c053214097009ccbde, e95983f0760e1ce53e1b36cf2f973f95fd4ed498, 51547a0e58fb8a8d3784a6dac5333de95ab61f63, 1bdbcdbb6592c7dfacea7196b5d58804151b1d61), which is why I really didn't want to merge changes that I thought made it worse. I might be wrong about on how to best structure the quickstart docs, but I definitely just want to make the docs as good as possible, and not trying to be a pain in the ass or anything :/.

Reply:
Oh yea, I wasnt thinking about the fact that you had been working on this recently. Then it makes more sense to be picky. Maybe next time when a documentation update is needed we can do it together or something.

Reply:
That sounds good! I do realize that all my comments did make me a pain in the ass, especially if you didn’t know I recently did a rewrite of that documentation page.